### PR TITLE
Rename Contact->Communication in UI <PD-1523>

### PR DESCRIPTION
**Pull request by tdphillips**
Tests pass.

### CODE REVIEW

File: templates/mypartners/view_record.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
             </div>
             {% endif %}
             <div class="tip-box" id="time">
-            <h5>Date & Time of Contact</h5>
+            <h5>Date & Time of Communication</h5>
```
Comment (tdphillips) [inline templates/mypartners/view_record.html]: This comes through just fine on the UI side. Leave as is or change to `&amp;`?

Comment (aspidites) [inline templates/mypartners/view_record.html]: @galitskyd , @fezick ?

Comment (galitskyd) [inline templates/mypartners/view_record.html]: Use &amp;

**Comment by tdphillips** [comment]
2015-07-30-092438_267x241_scrot
2015-07-30-092527_481x128_scrot
2015-07-30-092617_282x210_scrot
2015-07-30-092627_314x166_scrot

**Comment by aspidites** [comment]
@tdphillips @galitskyd , as for the weird wrapping, what about either:
1. Adding a few pixels in the css to allow for the text to not wrap
2. "Date & Time of Communication" -> "Date of Communication"
3. "Date & Time of Communication" -> "Date & Time"

**Pull request merged**